TransformationSystem/Agent/TaskManagerAgentBase.py

```diff
@@ -475,6 +478,9 @@ def submitTasks( self, transIDOPBody, clients ):
 
     return S_OK()
 
+  # This gSynchro is necessary in order to avoid race conditions when submitting to the WMS,
+  # because WMSClient wants jobDescription.xml to be present in the local directory prior to submission
+  @gSynchro
```

Comment:
This decorator (together with the comment above) was removed in previous version (e.g. latest pre-releases of v6r14), why is it back here?